recipes/flann/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set version = "1.8.4" %}
+
+package:
+    name: flann
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/mariusmuja/flann.git
+    git_tag: 40d63fc7db5d1f6dd67364f24e4579aedb57e7b6
+
+build:
+    number: 0
+    string: {{PKG_BUILDNUM}}_g{{GIT_FULL_HASH[:7]}}
```

Comment:
Ok, that's fine. We may make the feedstock (repo) have a stable release and a dev branch. That way people wanting a formal release can have it. This is done with different stable releases already so should be fine. There are some points about dev releases in general that haven't been discussed and maybe should be.